app/src/main/res/layout/onboarding_fragment.xml

```diff
@@ -54,6 +54,8 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="32dp"
         android:layout_marginBottom="16dp"
+        android:accessibilityTraversalAfter="@id/onboarding_fragment_next_image_view"
```

Comment:
I am not fully sure about this solution for ordering of output. I will research and get back.

Reply:
Okay. I'll wait for your response.

Reply:
Confirmed. We should not used `android:accessibilityTraversalAfter/Before`. Let's not try to solve this traversal order for now. I think we can keep the existing order for now.

Reply:
Okay. I'll remove it. Thanks.

Reply:
Done.